CI/setup-windows-sdk.sh

```diff
@@ -142,13 +142,25 @@ else
   done
 fi
 
+PACMAN_CONF="/etc/pacman.conf"
+
+# Use sed to uncomment the IgnorePkg line and set it to IgnorePkg = luajit
+sed -i '/^#IgnorePkg/s/^#//; s/^IgnorePkg.*/IgnorePkg = luajit/' "$PACMAN_CONF"
```

Comment:
What does this bit do?

Reply:
Oh, that probably isn't needed anymore.. I found that when doing a manual pacman -Su ccache it prompted me to replace the system lua (not the mingw-64-x86_64-lua51) with luajit, but in doing so it broke the luarocks build.

Reply:
Ah ok, let's remove it if not needed

Reply:
TBH I have a similar line in my own `/etc/pacman.conf` - effectively I've done:
```diff
 # Pacman won't upgrade packages listed in IgnorePkg and members of IgnoreGroup
-#IgnorePkg =
+IgnorePkg = mingw-w64-i686-lua51 mingw-w64-x86_64-lua51
 #IgnoreGroup =
```
for exactly the same reason - to avoid being updated from Lua 5.1 to LuaJit ...